test/toUinteger32.js

```diff
@@ -0,0 +1,73 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('toUinteger32', function () {
+  context('given a floating point number', function () {
+    specify('should drop decimal values', function () {
+      assert.strictEqual(RA.toUinteger32(123.456), 123);
+      assert.strictEqual(RA.toUinteger32(0.123456), 0);
+
+      assert.strictEqual(RA.toUint32(123.456), 123);
+      assert.strictEqual(RA.toUint32(0.12345), 0);
+    });
+
+    specify('should return an unsigned (non-negative) integer', function () {
+      assert.isAtLeast(RA.toUinteger32(-1.23456), 0);
+
+      assert.isAtLeast(RA.toUint32(-1.23456), 0);
+    });
+  });
+
+  context('given an integer', function () {
+    specify('should return the same value', function () {
+      assert.strictEqual(RA.toUinteger32(123), 123);
+      assert.strictEqual(RA.toUinteger32(0), 0);
+
+      assert.strictEqual(RA.toUint32(123), 123);
+      assert.strictEqual(RA.toUint32(0), 0);
+    });
+  });
+
+  context('given a non `number` type value', function () {
+    specify('should return an unsigned (non-negative) integer', function () {
+      assert.isAtLeast(RA.toUinteger32(true), 0);
```

Comment:
Is there a specific reason we're using `isAtLeast` here instead of `strictEqual` ?

Reply:
At first, I thought of using isAtLeast over strictEqual since I was unsure of the return value given a non-number type and that it would be valid anyways as long as it was an unsigned integer. I just recently looked at #783 and after doing a bit more research, I think it makes more sense for the types I listed there to return 0. 

One question, is it acceptable for the function to return 1 given a true value? Please let me know and I will make those changes!